src/components/TopNav.jsx

```diff
@@ -0,0 +1,206 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+import { createArticleIcon, notificationIcon } from '../assets/img__func/icons_svg';
+import Logo from './logo';
+import UserNavAvatar from './userNavAvartar';
+// import Hamburger from './Hamburger';
```

Comment:
@shaolinmkz, kindly remove the above commented code

Reply:
On it..